br/pkg/lightning/restore/restore.go

```diff
@@ -263,6 +264,8 @@ type ControllerParam struct {
 	CheckpointStorage storage.ExternalStorage
 	// when CheckpointStorage is not nil, save file checkpoint to it with this name
 	CheckpointName string
+	// DupIndicator can expose the duplicate detection result to the caller
+	DupIndicator *bool
```

Comment:
Maybe `atomic.Bool` is better. If we are importing multiple tables, this field may be set concurrently.